RecoParticleFlow/PFClusterProducer/plugins/PFClusterProducer.cc

```diff
@@ -45,6 +46,33 @@ class PFClusterProducer : public edm::stream::EDProducer<> {
 #include "FWCore/Framework/interface/MakerMacros.h"
 DEFINE_FWK_MODULE(PFClusterProducer);
 
+void PFClusterProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<edm::InputTag>("recHitsSource", {});
+  desc.add<bool>("usePFThresholdsFromDB", false);
+  edm::ParameterSetDescription psd;
+  psd.setUnknown();
```

Comment:
see https://github.com/cms-sw/cmssw/pull/45212/files#r1638142798

Reply:
The issue I had in this particular class was that there were so many variations of these parameter set, that failed to find a super set that cover all (so end up declaring it as unknown). Same with PFRecHitProducer

Reply:
The unknown parameter set descriptions are removed in 2da72f03d53a1bd5cb9fc6e0f72f9205dce0e900